src/ServiceManager/Travely.ServiceManager.Service/Managers/ActivityManager.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Travely.ServiceManager.Service.Managers
+{
+    public class ActivityManager : IActivityManager
+    {
+        public Task<int> CreateActivity(Activity activity)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<IEnumerable<Activity>> GetActivities(int agencyId)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<ActivityResponce> DeleteActivity(long activityId)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<ActivityResponce> EditActivity(Activity activity)
```

Comment:
Don't really think we need ActivityResponse here. These methods can get and return Activity model.